src/controllers/userconroller.js

```diff
@@ -75,12 +75,23 @@ export const updateStatus = catchAsync(async (req, res) => {
       },
       { where: { id } },
     );
-    await transporter.sendMail({
-      from: process.env.EMAIL_ADDRESS_STATUS,
-      to: user.email,
-      subject: `Your status has been updated to ${status}`,
-      text: `Dear ${user.name}, your status has been updated to ${status}.`,
-    });
+    if (user.status !== 'inactive') {
+      await transporter.sendMail({
+        from: process.env.EMAIL_ADDRESS_STATUS,
+        to: user.email,
+        subject: `Your status has been updated to ${status}`,
+        text: `Dear ${user.name}, due to violation of our policies, we regret to inform you that your account has been disactivated.`,
+      });
+    }
+    if (user.status === 'active') {
+      await transporter.sendMail({
+        from: process.env.EMAIL_ADDRESS,
+        to: user.email,
+        subject: `Your status has been updated to ${status}`,
+        text: `Hello ${user.name} hope this email finds you well, our team is pleased to inform you that your accout has been reactivated, your can now access all services we provide to you.`,
```

Comment:
Would mind to use a helper function for the email?

Reply:
yeah sure